pac4j-core/src/main/java/org/pac4j/core/context/WebContext.java

```diff
@@ -180,4 +182,12 @@ default void invalidationSession() {
      * @since 1.8.1
      */
     String getPath();
+
+    /**
+     * Gets content body of the original request.
+     *
+     * @return the request content
+     * @since 1.9.2
+     */
+    String getRequestContent();
```

Comment:
Yes, the idea was to provide a `default` implementation to keep backward compatibility (by returning an empty string).